manifests/rules/out/chrony.pp

```diff
@@ -1,7 +1,28 @@
-# manage out chrony
-class nftables::rules::out::chrony {
-  nftables::rule {
-    'default_out-chrony':
-      content => 'udp dport 123 accept',
+# @summary manage out chrony
+# @param servers single IP-Address or array of IP-addresses from NTP servers
+class nftables::rules::out::chrony (
+  Variant[Stdlib::IP::Address,Array[Stdlib::IP::Address]] $servers = [],
+) {
+  $_servers = [$servers].flatten
+  if empty($_servers) {
+    nftables::rule {
+      'default_out-chrony':
+        content => 'udp dport 123 accept',
+    }
+  } else {
+    $_servers.each |$index,$ps| {
+      nftables::rule {
+        "default_out-chrony-${index}":
+      }
+      if $ps =~ Stdlib::IP::Address::V6 {
```

Comment:
This does not follow https://puppet.com/docs/puppet/7/style_guide.html#style_guide_conditionals-simple-resource-declarations I would say.

Reply:
can you recheck this? I think it's fine now, since all the nftable::rule resources have different titles.